internal/net/call/call.go

```diff
@@ -843,7 +846,6 @@ func logError(logger *slog.Logger, details string, err error) {
 	if errors.Is(err, context.Canceled) ||
 		errors.Is(err, io.EOF) ||
 		errors.Is(err, io.ErrUnexpectedEOF) ||
-		errors.Is(err, io.ErrClosedPipe) ||
 		errors.Is(err, io.ErrClosedPipe) {
```

Comment:
This appeared to be checked twice, so I assume it's safe to remove one of them.

Reply:
Good catch.